### PR TITLE
TensorBoard 2.13.0

### PR DESCRIPTION
**Pull request by rileyajones**
## Motivation for features / changes
The TensorBoard release that tracks the TF 2.13 release.

## Technical description of changes
* Update TensorFlow protos as part of the 2.13 release.
* Pin the TF version for CI (to `tf-nightly=2.13.0.dev20230501`)

The proto updates were made by manually copying and editing the new file then updating the remaining files according to  [these intructions](https://github.com/tensorflow/tensorboard/blob/b6afe57af75636eaa61081f8f0e9636036aad4c8/tensorboard/compat/proto/proto_test.py)

### CODE REVIEW

File: tensorboard/version.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 """Contains the version string."""
 
-VERSION = "2.13.0a0"
+VERSION = "2.13.0"
```
Comment (arcra) [inline tensorboard/version.py:18]: We want each of these changes to be separate commits in the history of the release, that's why our release guide mentions that if they're all submitted in a single PR, we should use "rebase and merge" (referring to the fact that we would have multiple commits in the same PR).

See example from a previous release: https://github.com/tensorflow/tensorboard/pull/6192/commits

So we should have a commit with only the proto changes, and separate commits for the other steps of the release process.

Also, we will probably want these proto changes in the master branch, so you can choose to either submit those changes in master, and then create this branch again, or later cherry-pick those changes into master as well.

**Review by arcra — CHANGES_REQUESTED** [review]

File: tensorboard/version.py (hunk 1 above)
Comment (rileyajones) [reply tensorboard/version.py:18]: I didn't know about the convention of doing this all as a single PR with multiple ordered commits.

File: RELEASE.md [hunk 2]
```diff
@@ -592,7 +606,7 @@ The 2.1 minor series tracks TensorFlow 2.1.
   - In the event of a transient backend issue or permanent breaking change, the
     uploader can now gracefully degrade and print a diagnostic (#2879)
 
-[TensorBoard.dev]: https://tensorboard.dev/
+[tensorboard.dev]: https://tensorboard.dev/
```
Comment (arcra) [inline RELEASE.md:609]: I don't think this is correct. Maybe we should pull reformatting changes into a separate PR in the master branch, if you'd like? Let's just keep the new additions in this commit.

**Review by arcra — CHANGES_REQUESTED** [review]
Can you also add the "patch" version to the PR title? "Tensorboard 2.13.0"

**Review by arcra — APPROVED** [review]
